Design Style.css

```diff
@@ -0,0 +1,66 @@
+body {
+   /* background-image: url('natural.jpg');
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-size: cover;*/
+    margin: 0;
+}
+#container1{
```

Comment:
flow the cs conventions:  
1. add a line before and after the css selector
2. add a space between the selector namee and the right paranthisis
3. remove the empty line inside the selector area/scope